target/riscv/cheri-archspecific-early.h

```diff
@@ -34,7 +34,13 @@
 
 #include "cheri_defs.h"
 
-// Table 3.3: Capability Exception Codes
+/*
+ * TODO: this controls how many tags are fetched with cgettags/csettags. Check
+ * what the proper value is for RISCV.
+ */
+#define CAP_TAG_GET_MANY_SHFT 3
```

Comment:
Cache lines in our implementations are 64 bytes, giving 8 capabilities on RV32 and 4 capabilities on RV64 (latter being the same as Morello).

Reply:
Weird. It is strange that no RV64 tests failed with CAP_TAG_GET_MANY_SHFT=3 (which gives 8 caps loaded).

Reply:
Well I guess that's because we don't have any tests that run as part of CI?

Reply:
the cheribsd tests passed with it being 3. Running them now with 2 to be sure.

Reply:
We don't have a CLoadTags test in cheribsdtest, but even if we did it'd have to be somewhat generic. Technically for CHERI-RISC-V it's legal to have more than 4 capabilities on RV64, it's just whatever the cache line size gives you, but we make all our implementations the same so TestRIG doesn't find false positives.